packages/react-app/src/components/dashboard/DashboardMetrics.tsx

```
@@ -1,260 +1,369 @@
+  const totalBuyingPower = useAppStore(s => s.getTotalBuyingPower());
+  const riskFreeRate = useAppStore(s => s.riskFreeRate);
+  const analytics = useAppStore(s => s.analytics);
+  const isLoadingAnalytics = useAppStore(s => s.isLoadingAnalytics);
+  const analyticsError = useAppStore(s => s.analyticsError);
+  const fetchAnalyticsData = useAppStore(s => s.fetchAnalyticsData);
+  const currentViewParams = useAppStore(s => s.currentViewParams);
+
+  // Calculate Sharpe Ratio (annualized)
+  let sharpeRatio = 0;
+  let sortinoRatio = 0;
+  if (analytics && analytics.equityCurve && analytics.equityCurve.length > 1) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)